src/infra/controllers/submission.ts

```diff
@@ -0,0 +1,69 @@
+import * as submission from "../../domain/services/submission";
+import { Request, Response } from "express";
+import { TsvUtils, ControllerUtils } from "../utils";
```

Comment:
A benefit of using name spacing is avoiding these sort of imports. See my other comment about module vs namespace.